.markdownlint.yml

```diff
@@ -0,0 +1,166 @@
+# MD001/heading-increment/header-increment - Heading levels should only increment by one level at a time
```

Comment:
This config file doesn't include `default: true`, so only the markdownlint rules specified in it will be run. That means that as new versions of markdownlint get added (hopefully automatically from dependabot!) any new rules of theirs won't be enabled by default.

It's generally easier to go with the defaults as described in https://github.com/igorshubovych/markdownlint-cli#configuration:

```json
{
	"default": true
}
```

I switched the file to that locally, ran `npx markdownlint **/*.md`, and disabled all the rules reporting failures:

```json
{
	"default": true,
	"md010": false,
	"md013": false,
	"md033": false,
	"md038": false
}
```

Were there any other customizations you wanted to keep in here? Which brings up another downside of specifying all rules explicitly: it's hard to tell what's a default rule setting vs. an intentionally customized one.

Reply:
ah this is cleaner !, I like this 😅